Source/CovPassCheckApp/Source/Resources/Locale/en.lproj/Localizable.strings

```diff
@@ -11,210 +11,209 @@
+"validation_second_onboarding_page_message" = "As the person checking, you will only see the validity status of the digital COVID certificate and the name and date of birth of the certificate holder in the CovPassCheck app. Match this information with an identification document (e.g. ID card, passport). An active internet connection is not required for the check.";
 
-"validation_fourth_onboarding_page_title" = "Datenschutz und Datensicherheit";
+"validation_fourth_onboarding_page_title" = "Data protection and data security";
 
-"validation_fourth_onboarding_first_list_item" = "Die Überprüfung von QR-Codes anderer Personen und das Auslesen von Daten aus digitalen COVID-Zertifikaten ist nur in den gesetzlich vorgesehenen Fällen zulässig.";
+"validation_fourth_onboarding_first_list_item" = "The checking of QR codes of other persons and the reading of data from digital COVID certificates is only permitted in the cases provided for by law.";
 
-"validation_fourth_onboarding_second_list_item" = "Eine Speicherung, Weitergabe oder Nutzung dieser Daten zu anderen Zwecken ist unzulässig.";
+"validation_fourth_onboarding_second_list_item" = "Storage, disclosure or use of this data for other purposes is not permitted.";
 
-"validation_fourth_onboarding_third_list_item" = "Die Verwendung der digitalen COVID-Zertifikate ist freiwillig. Niemand darf zur Nutzung gezwungen werden. Der Nachweis eines vollständigen Impfschutzes, eines negativen Testergebnisses oder einer durchgemachten COVID-Infektion kann auch auf andere Weise erbracht werden.";
+"validation_fourth_onboarding_third_list_item" = "The use of COVID digital certificates is voluntary. No one may be forced to use them. Proof of complete vaccination protection, a negative test result or a COVID infection that has been passed through can also be provided by other means.";
 
-"vaccination_fourth_onboarding_page_message" = "Ausführliche Hinweise zur Datenverarbeitung finden Sie in der Datenschutzerklärung:";
+"vaccination_fourth_onboarding_page_message" = "Detailed information on data processing can be found in the privacy policy:";
 
-"confirmation_fourth_onboarding_page_button_title" = "Alles klar";
+"confirmation_fourth_onboarding_page_button_title" = "All clear";
 
-"vaccination_privacy_policy_page_button_title" = "Schließen";
+"vaccination_privacy_policy_page_button_title" = "Close";
 
-"error_connect_internet_for_scan_title" = "Flugzeugmodus ausschalten oder WLAN für den Datenzugriff verwenden";
+"error_connect_internet_for_scan_title" = "Turn off airplane mode or use Wi-Fi to access data";
 
-"error_connect_internet_for_scan_button_settings" = "Einstellungen";
+"error_connect_internet_for_scan_button_settings" = "Settings";
 
 "error_connect_internet_for_scan_button_ok" = "OK";
 
 
 /********** start screen **********/
 
-"validation_start_screen_title" = "Übersicht";
+"validation_start_screen_title" = "Overview";
 
-"validation_start_screen_scan_title" = "Zertifikat prüfen";
```

Comment:
Has this been removed on purpose?

Reply:
Oops, that wasn't intended. Thanks for noticing!